src/OpenSage.Game/LanguageSetting.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OpenSage.Data;
+using OpenSage.Utilities;
+
+namespace OpenSage
+{
+    public static class LanguageSetting
+    {
+        //do we need a default?
+        private const string DefaultLanguage = "englisch";
+        private static string _detectedLanguage;
+
+        public static string Current => string.IsNullOrEmpty(_detectedLanguage) ? DefaultLanguage : _detectedLanguage;
+
+        public static void ReadFromRegistry(IGameDefinition gameDefinition)
```

Comment:
You should add some additional logic to check if the current platform (`Environment.OSVersion`) is Windows or not. AFAIK calling any of the registry methods on Mac and Linux will throw `PlatformNotSupportedException`. If the game is not running on Windows, just default to english until we have a configuration option for this.